src/backend/gl/src/device.rs

```diff
@@ -998,7 +998,20 @@ impl d::Device<B> for Device {
                 }
 
             }
-            let _status = gl.check_framebuffer_status(target); //TODO: check status
+            
+            let status = gl.check_framebuffer_status(target);
+            match status {
+                glow::FRAMEBUFFER_COMPLETE => {},
+                glow::FRAMEBUFFER_INCOMPLETE_ATTACHMENT => panic!("One of framebuffer attachmet points are incomplete"),
+                glow::FRAMEBUFFER_INCOMPLETE_MISSING_ATTACHMENT => panic!("Framebuffer does not have any image attached"),
+                glow::FRAMEBUFFER_INCOMPLETE_DRAW_BUFFER => panic!("FRAMEBUFFER_INCOMPLETE_DRAW_BUFFER"),
+                glow::FRAMEBUFFER_INCOMPLETE_READ_BUFFER => panic!("FRAMEBUFFER_INCOMPLETE_READ_BUFFER"),
+                glow::FRAMEBUFFER_UNSUPPORTED => panic!("FRAMEBUFFER_UNSUPPORTED"),
```

Comment:
why can't we just `panic!("{:?}", status)` instead of repeating ourselves?

Reply:
Because you will get an integer